ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,34 @@
+<!--- Provide a general summary of the issue in the Title above -->
+
+## Your Environment
+<!--- Include as many relevant details about the environment you experienced the bug in -->
+* Version used:
+* Browser Name and version:
+* Operating System and version (desktop or mobile):
+* Link to your project:
+
+## Context
+<!--- Provide a more detailed introduction to the issue itself, and why you consider it to be a bug -->
+<!--- How has this bug affected you? What were you trying to accomplish? -->
+<!--- If you have errors or a stacktrace in the console, here is a good place
+to put it. Please consider using a pastebin service or create a gist.-->
```

Comment:
nit: strange to use a multiline comment here; inconsistent with the rest of the template (e.g. steps to reproduce)